README.md

```diff
@@ -1,20 +1,210 @@
+
+### `poe db-test`
+
+Run `poe db-test` to run helper functions for the test database. See more about it in Database Management.
+
+### `poe dev`
+
+Run `poe dev` to start the database, a celery-worker, a redis database, and the development server. You can access the endpoint of the REST-API at port 5010.
+
+### `poe prod`
+
+Run `poe prod` to start the database, celery-worker, a redis database, and the waitress production server. You can access the endpoint of the REST-API at port 8090.
+
+### `poe gui`
+
+Run `poe gui` to start the database and the development server. You can access the endpoint of the REST-API at port 5010. This command disables the celery worker and enables the GUI representation of the simulation.
+
+### `poe insert-config`
+
+Run `poe insert-config` to insert a previously selected config. By default this is one of the configs in `/scripts`.
 
 ## Environment variables
 
-We're using `dotenv` to load environment variables. We define with `poe` some handy commands for development. Please take a look at the documentation in the [wiki](https://github.com/BP2022-AP1/bp2022-ap1/wiki#environment-variables).
-## poe
+We're loading public environment variables with docker compose and secret ones with poe. We have four files that contain environment variables.
+
+### `.env.shared`
+
+This file contains harmless public environment variables. They are used by every deployment (test, dev, prod) but can be overriden.
+
+### `.env.test`
+
+This file contains public environment variables for the test environment.
+
+### `.env.dev`
+
+This file contains public environment variables for the dev environment.
+
+### `.env.secret`
+
+This file contains secret variables for the production environment, that shouldn't be shared. It overrides variables from `.env.shared`, `.env.shared` and `os`.
+
+### Important environment variables
+
+- `DISABLE_CELERY` - This variable disables running the simulation in a celery worker and enables the GUI representation of the simulation
+- `SUMO_DELAY` - This variable edits the delay time for the GUI representation of the simulation, enabled by `DISABLE_CELERY`.
+
+
+
+## PlanPro and SUMO configuration Generator
+
+### PlanPro
+
+There is a ```planpro_helper.py``` in the ```data``` directory, which main method uses yaramo to get a PlanPro file from OSM with hardcoded coordinates. The result is saved into the ```planpro``` subdirectory with a name chosen in ```planpro_helper.py```.
+
+### SUMO configuration
+
+There is a ```sumo_config_helper.py``` in the ```data``` directory, which main method uses yaramo to get a sumo-config from a PlanPro file. The name can not be chosen, but is the one from the input PlanPro file. There is a directory inside the ```sumo``` subdirectory with that name. Inside there is the ```sumo-config```directory with the SUMO files.
+
+By running `bin/db` you can manage the `dev` database and the `test` database.
```

Comment:
There seems to be no `bin/db` anymore. Do you mean `db/local/db.py`?

Reply:
That is correct :D